packages/repository/src/relations/has-many/has-many.decorator.ts

```diff
@@ -21,7 +21,7 @@ export function hasMany<T extends Entity>(
   definition?: Partial<HasManyDefinition>,
 ) {
   return function(decoratedTarget: Object, key: string) {
-    property.array(targetResolver)(decoratedTarget, key);
+    //property.array(targetResolver)(decoratedTarget, key);
```

Comment:
You can remove this line 👍

Reply:
Hi @b-admike ,
thanks for your help.
i opened a new PR included the fix PR #2178